docs/external_resources.md

```diff
@@ -96,6 +96,20 @@ Links: [Blog Post](https://blog.holoviz.org/posts/tweak-mpl-chat/) | [Hugging Fa
 
 ![Talk to Plot](https://blog.holoviz.org/posts/tweak-mpl-chat/images/app1.gif)
 
+## OpenAI
+
+### News Summarizer
+
```

Comment:
[news-summarizer-demo.zip](https://github.com/holoviz-topics/panel-chat-examples/files/14149656/news-summarizer-demo.zip)

feel free to include the recording here!